src/common/ZulipButton.js

```diff
@@ -121,54 +121,54 @@ type Props = $ReadOnly<{|
  * @prop [secondary] - Less prominent styling, the button is not as important.
  * @prop onPress - Event called on button press.
  */
-export default class ZulipButton extends PureComponent<Props> {
-  static defaultProps = {
-    secondary: false,
-    disabled: false,
-    progress: false,
-  };
-
-  render() {
-    const { style, text, disabled, secondary, progress, onPress, Icon } = this.props;
-    const frameStyle = [
-      styles.frame,
-      // Prettier bug on nested ternary
-      /* prettier-ignore */
-      disabled
+export default function ZulipButton(props: Props): React$Node {
+  const {
+    style,
+    text,
+    disabled = false,
+    secondary = false,
+    progress = false,
+    onPress,
+    Icon,
+  } = props;
+  const frameStyle = [
+    styles.frame,
+    // Prettier bug on nested ternary
+    /* prettier-ignore */
+    disabled
         ? secondary
           ? styles.disabledSecondaryFrame
```

Comment:
nit: dedent the interior of this expression, along with the rest of the code

(Prettier doesn't take care of it because of that prettier-ignore above, but it sticks out in the diff)